inc/class-wpseo-utils.php

```diff
@@ -61,7 +61,15 @@ public static function is_apache() {
 
 		$software = sanitize_text_field( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) );
 
-		return stripos( $software, 'apache' ) !== false;
+		if ( stripos( $software, 'apache' ) !== false ) {
+			$abool = true;
+		}
+		elseif ( stripos( $software, 'litespeed' ) !== false ) {
+			$abool = true;
+		}
+		else $abool = false;
```

Comment:
PHPCS failed because "else" isn't enclosed in the curly braces. Please check travis log.